openshift/templates/icbc-web-adapter.dc.yaml

```diff
@@ -126,20 +126,20 @@ objects:
                   value: guest
                 - name: RABBITMQ__PASSWORD
                   value: guest
-                - name: ICBCSOAPAPI__BASEADDRESS
+                - name: ICBCRESTADAPTER__BASEURL
                   valueFrom:
                     secretKeyRef:
-                      name: icbc-soap-api-configuration
-                      key: base-address
-                - name: ICBCSOAPAPI__USERNAME
+                      name: icbc-rest-adapter-configuration
+                      key: base-url
+                - name: ICBCRESTADAPTER__USERNAME
```

Comment:
We do not need ICBCRESTADAPTER__USERNAME

Reply:
fixed